README.md

```diff
@@ -1088,6 +1088,44 @@ html {
 ```
 
 
+## Lambda functions
+
+### Events emitted by the library
+
+* `page_pre_render`: When a page is loaded and about to be rendered, will contain the response from your section page as a payload
+
+### Functions you can call
+
+Inside `sections/js/global-hooks.js`
+
+* `page_pre_load`: When a page is about to be loaded, it allow you to update the payload sent for the render page API request
+
+* `section_pre_render`: When a view section component is being picked up for rendering (before rendering), it allow to change the view component before rendering it
```

Comment:
You don't call these functions, these are callbacks you can implement, isn't it ?